lib/puppet/provider/cisco_bgp_af/nxapi.rb

```diff
@@ -53,6 +53,7 @@
   ]
 
   BGP_AF_BOOL_PROPS = [
+    :advertise_l2vpn_evpn,
```

Comment:
Please keep this in alpha order unless there's a dependency reason to place this property here.